### ISSUE TITLE
Navigational keys not repeating when holding down.

### ISSUE THREAD

**Issue by tsteuwer-accesso** [VSCodeVim/Vim#4608]
**Describe the bug**
Navigational keys are not repeating when holding them down.

**To Reproduce**

1.  Go to any file.
2. Use navigational keys (j, k, l, h) and hold them down.

**Expected behavior**
The cursor will continue to go in the direction pressed until I let go of the key.

**Screenshots**
N/A

**Environment (please complete the following information):**

- Extension (VsCodeVim) version: v1.13.0
- VSCode version: 
```
Version: 1.42.1
Commit: c47d83b293181d9be64f27ff093689e8e7aed054
Date: 2020-02-11T14:44:27.652Z
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 19.3.0
```
- OS: MacBook Pro (16 inch, 2019)
  macOS 10.15.3

**Additional context**
When in insert mode or in other terminals, the keys repeat.

**Comment by schu34** [comment]
Have you tried these instructions? https://github.com/VSCodeVim/Vim#mac
tldr run this in your terminal and restart vscode
```
$ defaults write com.microsoft.VSCode ApplePressAndHoldEnabled -bool false         # For VS Code
```

**Comment by tsteuwer-accesso** [comment]
That didn't work, but I did find out after updating the repeat keys option was turned off. This can be closed. Thanks for trying to assist!

**Comment by luciferkonn** [comment]
Hi @tsteuwer-accesso, I got the same issue. Could you elaborate more on how you solved this issue? Thanks!
